.claude/skills/tdd-methodology/SKILL.md

```
@@ -111,12 +111,27 @@ tests/
 ```
 
 ### Frontend Tests (Vitest)
+
+テスト対象と同じ場所に `*.test.tsx` / `*.test.ts` を置く。
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)